src/main/webapp/views/editCard.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>Vendr</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <!-- Dependencies -->
+  <link rel="stylesheet" href="../styles/bootstrap.min.css">
+  <link rel="stylesheet" href="../font-awesome-4.7.0/css/font-awesome.min.css">
+  <script src="../scripts/jquery-3.5.1.min.js"></script>
+  <script src="../scripts/popper.min.js"></script>
+  <script src="../scripts/bootstrap.bundle.min.js"></script>
+  <script src="../scripts/script.js"></script>
+  <script src="../scripts/navbar.js"></script>
+  <!--Mock vendor object -->
+  <script src="mockSingleVendor.js"></script>
+  <script src="../scripts/editCard.js"></script>
+</head>
+
+<body>
+  <div id="navbar_container"></div>
+  <div id="card-placeholder" class="container mb-5">
+    <div class="row justify-content-md-center mt-3">
+      <div class="col-12">
+        <div class="card border-primary">
+          <h5 class="card-header">Edit card #ID</h5>
+          <div class="card-body">
+            <div id="card-form">
```

Comment:
I don't see you are using a proper `form` element, I think I commented something similar in another PR but it wasn't addressed.

Reply:
> I guess it is the same as before, you are going to submit through Javascript, right?
My only concern is with accessibility, since you are not using a `<form>`, which is already a semantic element. 

I guess we can ask during the standup meeting.